src/v2/Apps/PriceDatabase/PriceDatabase.tsx

```diff
@@ -11,11 +11,13 @@ import { ActionType, ContextModule, OwnerType } from "@artsy/cohesion"
 export const PriceDatabase = () => {
   const { trackEvent } = useTracking()
 
-  trackEvent({
-    action: ActionType.screen,
-    context_module: ContextModule.priceDatabaseLanding,
-    context_screen_owner_type: OwnerType.priceDatabase,
-  })
+  useEffect(() => {
+    trackEvent({
+      action: ActionType.screen,
+      context_module: ContextModule.priceDatabaseLanding,
+      context_screen_owner_type: OwnerType.priceDatabase,
+    })
+  }, [])
```

Comment:
@MounirDhahri I'm not quite sure if this is the correct way to do screen tracking.

Reply:
this is a good catch
I am also not sure if that's the way to do screen tracking here but I think's fine for now

Reply:
@damassi is there a better way to do screen tracking in Force?